packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -340,6 +376,7 @@ export class ChildLogger extends TelemetryLogger {
 		);
 	}
 
+	private readonly [childLoggerType] = childLoggerType;
```

Comment:
I'm assuming this is to allow ChildLogger coallescince across layers at different versions where `instanceof` doesn't work?  Maybe put in a different PR